packages/ui/src/components/va-data-table/VaDataTable.demo.vue

```diff
@@ -483,9 +482,9 @@
       <va-data-table :items="evenItemsShuffled" :columns="evenColumnsSortable" v-model:sort-by="sortByModel" v-model:sorting-order="sortingOrderModel" />
     </VbCard>
 
-    <VbCard title="Paginationser" class="demo">
```

Comment:
Thanks for snagging the spelling changes. These were intentionally to try and figure out which demo was being used so I could watch for a console log and I was planning to revert them once I saw the event was emitting correctly.

Reply:
I made new demo in strorybook. We're slowly moving from old demo format to new one, so I reverted previous changes, because we will probably remove these demos in future anyway.

Reply:
Ah okay. How do I test demos in storybook?

Reply:
run `yarn serve:storybook`

visit http://localhost:6006/?path=/story/vadatatable--column-sorted-event